src/Microsoft.VisualStudio.Editors/SettingsDesigner/TypePickerDialog.vb

```diff
@@ -446,15 +447,19 @@ Namespace Microsoft.VisualStudio.Editors.SettingsDesigner
                 Dim assemblyImage As Bitmap = GetManifestBitmapTransparent("assembly.bmp", Color.FromArgb(255, 0, 255))
                 Dim namespaceImage As Bitmap = GetManifestBitmapTransparent("namespace.bmp", Color.FromArgb(255, 0, 255))
                 Dim objectImage As Bitmap = GetManifestBitmapTransparent("object.bmp", Color.FromArgb(255, 0, 255))
+                'Scale the images for per monitor DPI
+                Handle.LogicalToDeviceUnits(assemblyImage.Width)
```

Comment:
What exactly are these lines doing? You aren't using the result.

Reply:
Looking at the internal email it states:

> You should use the Visual Studio image service to replace your usage of DpiHelper’s image scaling methods.

Reply:
I believe the recommendation is to use image monikers throughout instead of scaling bitmaps manually.

Reply:
If we can find an existing bitmap here: https://glyphlist.azurewebsites.net/knownmonikers/ we can just use its moniker instead

Reply:
@jmarolf that's a great link, thanks!